src/workerd/api/queue.c++

```diff
@@ -504,10 +512,25 @@ jsg::Ref<QueueEvent> startQueueEvent(EventTarget& globalEventTarget,
     }
     globalEventTarget.dispatchEventImpl(lock, event.addRef());
     event->setCompletionStatus(QueueEvent::CompletedSuccessfully{});
+    if (FeatureFlags::get(js).getQueueConsumerNoWaitForWaitUntil()) {
+      // TODO(now): It doesn't really make sense for event-listener scripts NOT to wait on
```

Comment:
This is the biggest question here, and needs a decision before releasing this.

Reply:
(an important subtext here being that it's unlikely the `addEventListener` format is used much, so minimizing the effort involved in this is valuable)

Reply:
I don't have strong feelings on this one FWIW -- I tend to agree that the `addEventListener` format is very stale/unlikey-to-be-used (for Queue consumers at least) and not worth too much of your effort!

Reply:
hmm.. not sure I have much of an opinion on this one. Maybe @kentonv ?

Reply:
Thanks for chiming in!

FWIW the consensus amongst the team this morning was just to "ignore the compat flag and run with the old behavior" since we've never heard of anyone actually using service worker syntax for queues.